x-pack/plugins/cases/public/components/all_cases/columns.tsx

```diff
@@ -107,15 +105,17 @@ export const useCasesColumns = ({
   );
 
   const handleDispatchUpdate = useCallback(
-    (args: Omit<UpdateCase, 'refetchCasesStatus'>) => {
-      dispatchUpdateCaseProperty({
-        ...args,
-        refetchCasesStatus: () => {
+    ({ updateKey, updateValue, caseData }: UpdateByKey) => {
```

Comment:
This is one of the main logic-wise changes in this PR.  The previous useGetCases hook was providing a "sub-hook" to update a case and I had to remove that.

Instead I reused the existing updateCaseproperty hook.